studio/rabbit_queue.py

```diff
@@ -337,7 +376,7 @@ def stop(self):
         """
         self._logger.info('stopping')
         self._stopping = True
-        self.close_channel()
+        self._delete_queue()
```

Comment:
Can we leave the stop implementation as it standards and simply add a new public delete method so that the semantics of the exist method dont change. To comply to the open for extension closed for modification rule?

Given the public shutdown method is new this behavior could be added there.